lib/phoenix_live_view/channel.ex

```diff
@@ -1391,4 +1421,19 @@ defmodule Phoenix.LiveView.Channel do
   end
 
   defp maybe_subscribe_to_live_reload(response), do: response
+
+  defp component_privates(state) do
```

Comment:
you only call this [function once](https://github.com/phoenixframework/phoenix_live_view/pull/2763/files#diff-e9718fd7326b6d6cbb34298f653b8edace4a97afe17389509dc833621fdcd3d6R297), should it maybe be component_pids and return the pids directly?